lib/collective/collectors/trackjs.rb

```diff
@@ -0,0 +1,110 @@
+require 'pp'
+
+module Collective::Collectors
+  class TrackJS < Collective::Collector
+    requires :faraday
+    requires :faraday_middleware
+    requires :json
+
+    resolution '60s'
+
+    def initialize(lastSeenId = nil, options)
+      @lastSeenId = lastSeenId
+      super(options)
+    end
+
+    collect do
+      instrument_errors
+    end
+
+    private
+
+    def instrument_errors
+      count = 0
+      application = ""
+      paged("/#{customer_id}/v1/errors").each do |error|
+        application = error['application'] if count == 0
+        count += 1
+      end
+      instrument 'trackjs.url.errors', count, source: application, type: 'count'
```

Comment:
Ok, yeah then I imagine it might also be useful to break the errors down by application and browser name/version. Take a look at https://github.com/remind101/collective/blob/master/lib/collective/collectors/postgres.rb#L22-L23.

This allows some powerful visualizations like the last last four graphs here: https://app.datadoghq.com/dash/87613/whats-happening-w-prod-api-db?live=true&page=0&is_auto=false&from_ts=1461884463516&to_ts=1462489263516&tile_size=m